Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -37,6 +37,21 @@ protected function getTestFile($filename = __FILE__)
         return $files[$filename];
     }
 
+    /**
+     * Tests if a fixer fixes a given input string to match the expected result.
+     *
+     * It is used both if you want to test if something is fixed or if it is not touched by the fixer.
```

Comment:
I updated it to obey the restrictions, although 80 characters seems not much. Have a look and decide.